sleap/nn/tracking.py

```diff
@@ -763,6 +775,30 @@ def int_list_func(s):
         ] = "For Kalman filter: Number of frames to track with other tracker. 0 means no Kalman filters will be used."
         options.append(option)
 
+        def float_list_func(s):
+            return [float(x.strip()) for x in s.split(",")] if s else None
+
+        option = dict(name="oks_errors", default="")
```

Comment:
I tried several videos with different zoom values and I find 5 is a better default value. I only have videos with two mice, but it would be worth checking the `metrics.val.npz` of other animals and points of view, to see if it makes sense to put 5 as default instead of 1.
The code to get the error from the validation set on each keypoint:

```python
p = "metrics.val.npz"
a = np.load(p, allow_pickle=True)
ds = a["metrics"][()]["dist.dists"]
np.nanmean(ds, axis=0)
```